src/app/modules/main/chat_section/module.nim

```diff
@@ -889,9 +890,12 @@ method onCommunityCheckAllChannelsPermissionsResponse*(self: Module, checkAllCha
 
 method onKickedFromCommunity*(self: Module) =
   self.view.setAmIMember(false)
+  let communityId = self.controller.getMySectionId()
+  self.view.setIsControlNodeOffline(self.controller.isCommunityRequestAwaitingAddress(communityId))
 
 method onJoinedCommunity*(self: Module) =
   self.view.setAmIMember(true)
+  self.view.setIsControlNodeOffline(false)
```

Comment:
I understand why you use `isControlOffline` for the status and functions, because that way we can use it for other scenarios as well and it's more generic, but since we can never really know if the control node is online or offline, I wonder if this will come back with side effects later.

I'm not really asking for change, just curious if there is a better way. 

For example here, you infer from the fact that we joined the community that the control node is back online, but that's the only place we set offline to false, so maybe it will become confusing?

Reply:
renamed to be more specific